Misc/NEWS.d/next/Tests/2018-01-27-13-21-39.bpo-25095.s-xRa5.rst

```diff
@@ -0,0 +1,2 @@
+RequestHandlerLoggingTestCase.test_get now sends a "Connection" header with
```

Comment:
We don't require a NEWS entry for this issue, so I just set the "skip news" label. But it would be nice if we convert this into a comment in the test:

```py
self.con.request('GET', '/', headers={'Connection': 'close'})
```

Reply:
Unfortunately as stated in the linked BPO issue, the test hangs because the socket remains open and readable/writable as per HTTP 1.1, the test hangs as it awaits for the socket to terminate.